File: scripts/check_sdk.py
from claude_agent_sdk import ClaudeAgentOptions
opts = ClaudeAgentOptions(
    cli_path="/home/box/.local/bin/claude",
    cwd="/workspace/hello-cc",
    system_prompt={"type": "preset", "preset": "claude_code"},
    tools={"type": "preset", "preset": "claude_code"},
    include_hook_events=True,
    forward_subagent_text=True,
    permission_mode="default",
    setting_sources=["user", "project", "local"],
)
print("ok_options", type(opts).__name__, bool(opts.tools), opts.setting_sources)

File: scripts/check_sdk.ts
import { query } from "@anthropic-ai/claude-agent-sdk";

const opts = {
  pathToClaudeCodeExecutable: "/home/box/.local/bin/claude",
  cwd: "/workspace/hello-cc",
  systemPrompt: { type: "preset" as const, preset: "claude_code" as const },
  tools: { type: "preset" as const, preset: "claude_code" as const },
  includeHookEvents: true,
  forwardSubagentText: true,
  permissionMode: "default" as const,
  settingSources: ["user", "project", "local"] as const,
};
console.log("ok_options", typeof query, Boolean(opts.tools), opts.settingSources);
